__init__.py

```diff
@@ -261,13 +275,34 @@ def onBackgroundColor(self, context):
     # # Setup > Models > Display Pigtails
 
     def onDisplayPigtails(self, context):
-        self.updateDisplayPigtails()
+        for fixture in self.fixtures:
+            for obj in fixture.collection.objects:
+                if "Pigtail" in obj.name:
+                    obj.hide_set(not self.display_pigtails)
 
     display_pigtails: BoolProperty(
         name = "Display pigtails",
         default = False,
         update = onDisplayPigtails)
 
+    # # Logging levels
+
+    def onLoggingLevel(self, context):
+        DMX_LOG.log.setLevel(self.logging_level)
+
+    logging_level: bpy.props.EnumProperty(
+        name= "Logging level",
+        description= "logging level",
+        default = "ERROR",
+        items= [
+                ('CRITICAL', "Critical", ""),
```

Comment:
I *think* the last argument here refers to the icon, you might want to play around with that and see if it looks cool (sorry if you knew that already).

Reply:
Interesting, thank you. I added some icons, it is not easy to find good ones, feel free to suggest other ones :)

Reply:
I usually look at Blender's Icon Viewer:

![image](https://user-images.githubusercontent.com/2349393/214826982-de5841c3-093d-407e-902a-afd06898e1b6.png)

![image](https://user-images.githubusercontent.com/2349393/214827107-937b0687-c7e3-4908-87a5-5f7676124045.png)

![image](https://user-images.githubusercontent.com/2349393/214827168-900e8236-016b-4038-b3d9-2f079fdb35ca.png)

Reply:
Thank you!
Yes, i found it a day later and used it to improve the icons already :)